values/graphprotocol-indexer-agent.yaml

```diff
@@ -0,0 +1,23 @@
+role: indexer-agent
+
+postgres:
+  db: graph-node 
```

Comment:
So, all these components connect to the same DB, right?

Reply:
Yep, it can use same DB (as now) or it can use separate DB